modules/example/src/main/scala/example/RepositoriesElasticsearch.scala

```diff
@@ -32,7 +32,9 @@ import zio.prelude.Newtype.unsafeWrap
 final case class RepositoriesElasticsearch(executor: ElasticExecutor) {
 
   def findAll(): Task[List[GitHubRepo]] =
-    executor.execute(ElasticRequest.search[GitHubRepo](Index, matchAll))
+    executor
+      .execute(ElasticRequest.search[GitHubRepo](index = Index, query = matchAll, maybeAggregation = None))
```

Comment:
Could this be renamed to just `aggregation`, and have `None` as the default parameter? Then we would use a method on `Search` requests to specify it.